front/src/modules/study/components/AddOrEditStudyModal.tsx

```diff
@@ -16,10 +16,11 @@ import { RiCalendarLine, RiMoneyEuroCircleLine, RiQuestionLine } from 'react-ico
 import { useDispatch } from 'react-redux';
 import { useNavigate, useParams } from 'react-router-dom';
 import { setFailure, setSuccess } from 'reducers/main';
-import { updateStudyID } from 'reducers/osrdconf';
 import { StudyCreateForm, StudyWithScenarios, osrdEditoastApi } from 'common/api/osrdEditoastApi';
 import { STUDY_STATES, studyStates, studyTypes } from 'applications/operationalStudies/consts';
 import { formatDateForInput, getEarliestDate } from 'utils/date';
+import { useOsrdActions } from 'common/osrdContext';
+import { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
```

Comment:
Could you add `type` to all other type imports of the file please :)
```suggestion
import type { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
```

Reply:
Done